deeprank/features/BSA.py

```diff
@@ -120,9 +120,15 @@ def get_contact_residue_sasa(self, cutoff=5.5):
             atcenter = 'CB'
             if res[2] == 'GLY':
                 atcenter = 'CA'
-            xyz = self.sql.get(
-                'x,y,z', resSeq=res[1], chainID=res[0], name=atcenter)[0]
-            # xyz = np.mean(self.sql.get('x,y,z',resSeq=r[1],chainID=r[0]),0)
+                
+            try :
+                xyz = self.sql.get(
+                    'x,y,z', resSeq=res[1], chainID=res[0], name=atcenter)[0]
+            except IndexError :
+                warnings .warn('Atom ', atcenter, ' not found for residue ', key[1], \
```

Comment:
need to remove the space between `warnings` and `.warn`

Reply:
thanks